### ISSUE TITLE
Not appplicable/not recorded option for the Tag (protein) metadata

### ISSUE THREAD

**Issue by yukie-a** [glygener/glygen-array-backend#96]
**Is your feature request related to a problem? Please describe.**
It will be nice to have 'not applicable / not recorded' option for the tag information of a Sample (protein).

**Page**
Sample -> Protein -> Tag

**Comment by ReneRanzinger** [comment]
The Tag information (as a whole) is optional. Are we talking about the position field?

**Comment by yukie-a** [comment]
Yes, the tag section is optional. but in this  case, we know the the tag is 'FC-tag' but we do not know the position of tag added.. so we had to give up entering tag information... 

So this is not a 'bug' - but it will be nice to have this option to be able to enter the information as much as we can provide!

**Comment by ReneRanzinger** [comment]
@senaarpinar if we make this change in the schema whould that cause problems? Its not use yet anyway.
